rikaicontent.js

```diff
@@ -314,12 +319,28 @@ var rcxContent = {
 	},
 	
 	keysDown: [],
+	lastPos: { x: null, y: null},
+	lastTarget: null,
 
 	onKeyDown: function(ev) { rcxContent._onKeyDown(ev) },
 	_onKeyDown: function(ev) {
-//		this.status("keyCode=" + ev.keyCode + ' charCode=' + ev.charCode + ' detail=' + ev.detail);
-
-		if ((ev.altKey) || (ev.metaKey) || (ev.ctrlKey)) return;
+		//console.log("key=" + ev.key + " keyCode=" + ev.keyCode + ' charCode=' + ev.charCode + ' detail=' + ev.detail);
+ 
+		if (window.rikaichan.config.showOnKey !== "" && (ev.altKey || ev.ctrlKey)) {

```

Comment:
instead of checking if showOnKey is "", this seems like it would be better to store the keycode of the whichever key was chosen in options right?